biztalk/core/feature-pack-create-application.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Operation Name : Create BizTalk application
+description: In Azure DevOps, create a release definition to deploy the BizTalk Server application
+ms.custom: ""
+ms.date: "9/20/2022"
+ms.prod: "biztalk-server"
+ms.reviewer: ""
+
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+author: MandiOhlinger
+ms.author: abkumar
+manager: "anneta"
+---
+
+
+# OperationName: Create BizTalk application
+
+
+### Create new BizTalk Server Appication
+    :::image type="content" source="./media/feature-pack-add-build-release-definitions/azure-devops-create-operation.PNG" alt-text="Create BizTalk Server application as a deployment task for Azure DevOps in Visual Studio.":::

```

Comment:
there seems to be some formatting issue here:
![image](https://user-images.githubusercontent.com/45184761/191840497-4b31e15b-6ccf-46ca-9adc-ece0c6f18885.png)